tools/ci_testing/install_py36.sh

```diff
@@ -25,6 +25,7 @@ apt-get -y -qq update && apt-get -y -qq install python3.6
 
 python3.6 get-pip.py -q
 python3.6 -m pip --version
+rm get-pip.py
 
 ln -sfn /usr/bin/python3.6 /usr/bin/python3
 pip3 install scipy  # Pre-installed in custom-op
```

Comment:
Perhaps we could add scipy dependency in build_deps/requirements.txt and requirements_gpu.txt?

Reply:
I'm open to it, but it's only required in one test script. Custom-op has it pre-installed (I assume because it's a dependency of TF -- though that may not be true for recent versions)

Reply:
Thank you for clarification, Sean, I understand.